doc/nrf/releases/release-notes-2.3.0.rst

```diff
@@ -0,0 +1,1034 @@
+.. _ncs_release_notes_230:
+
+|NCS| v2.3.0 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, nRF70, and nRF91 Series.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+The following list includes the summary of the most relevant changes introduced in this release.
+
+* Added :ref:`support <software_maturity>` for the following features:
+
+  * nRF7002 Wi-Fi 6 companion IC and nRF7002 DK.
+  * Bluetooth® Low Energy: Period Advertising Sync Transfer.
+    This feature allows Periodic Advertising synchronization data to be transferred over an ACL connection, which gives power saving benefits for energy constraint devices.
+    This is a key feature for the `Auracast™`_ Assistant.
+  * nRF Cloud Location Services Wi-Fi® feature support in the :ref:`lib_location` library and demonstrated in :ref:`asset_tracker_v2`, :ref:`modem_shell_application`, and :ref:`lwm2m_client`.
+  * Writing modem trace to external flash for later retrieval.
+    The :ref:`modem_shell_application` sample now demonstrates how to store and upload modem trace to cloud.
+  * New :ref:`Serial LTE Modem (SLM) Shell <slm_shell_sample>` sample:
+    The sample demonstrates sending AT commands to the nRF9160 SiP from shell for nRF53 and nRF52 Series SoCs.
+  * New :ref:`mqtt_sample` sample supporting Wi-Fi together with cellular connectivity (replacing the Simple MQTT sample).
+
+* Added :ref:`experimental support <software_maturity>` for the following features:
+
+  * Periodic Advertisement with Responses (PAwR) - Advertiser.
+    This is a new feature introduced in Bluetooth v5.4 specification, enabling the bidirectional exchange of application data using connectionless communication.
+    Learn more in our `Bluetooth 5.4 DevZone blog`_.
+    The SoftDevice Controller continues to be Bluetooth v5.3 qualified.
+  * Bluetooth LE Audio: Public Broadcast Announcement (PBA) and bidirectional streams to and from two headsets.
+  * New :ref:`wifi_sr_coex_sample` sample.
+
+* Improved:
+
+  * The :ref:`matter_lock_sample` sample has been extended to support switching between Matter over Thread and Matter over Wi-Fi during application operation.
+  * The :ref:`wifi_shell_sample` sample by adding the Power Save feature.
+
+See :ref:`ncs_release_notes_230_changelog` for the complete list of changes.
+
+Sign up for the `nRF Connect SDK v2.3.0 webinar`_ to learn more about the new features.
```

Comment:
I would add these lines to line 20. Adding them there won't hide the highlights, but will bring more attention to the webinar and the changelog.